pkg/platform/authentication/auth.go

```diff
@@ -189,6 +189,9 @@ func (s *Auth) AuthenticateWithModel(credentials *mono_models.Credentials) error
 		case *apiAuth.PostLoginRetryWith:
 			return errs.AddTips(&ErrTokenRequired{locale.WrapInputError(err, "err_auth_fail_totp")}, tips...)
 		default:
+			if os.IsTimeout(err) {
+				return locale.NewInputError("err_api_auth_timeout", "Timed out waiting for authentication response. Please try again.")
+			}
```

Comment:
@Naatan This feels like it's stretching the idea of InputError. Is this OK?

Reply:
I think this is a valid input error. Input error for us basically means an error due to a user-side variable that we simply cannot address ourselves.